client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/hooks/useNextItem.test.tsx

```diff
@@ -81,7 +81,6 @@ const getLines = (): InvoiceLineNode[] => [
         averageMonthlyConsumption: 0,
       },
       atcCategory: '',
-      availableStockOnHand: 0,
```

Comment:
offroad to allow ts validation

Reply:
Ha, I just fixed this in #1171 as well ;)